src/Common/test/Common.TestResources/XunitLoggerProvider.cs

```diff
@@ -0,0 +1,102 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the Apache 2.0 License.
+// See the LICENSE file in the project root for more information.
+
+#nullable enable
+
+using Microsoft.Extensions.Logging;
+using Xunit.Abstractions;
+
+namespace Steeltoe.Common.TestResources;
+
+/// <summary>
+/// Writes log messages to XUnit output.
+/// </summary>
+public sealed class XunitLoggerProvider : ILoggerProvider
```

Comment:
Actually, I moved the existing one from Management and enhanced it a little bit, so it can filter on namespaces and levels (similar to `CapturingLoggerProvider`, which I added to assert on log output). The library looks nice, it adds syntactic sugar, custom formatters, etc. For our needs, it looks like overkill, so I prefer to keep our simple implementation and not take a dependency on it. The linked issue seems to be two-fold:

> handling logger output (for troubleshooting)

and

> validation (of framework code) in tests

Since both are available now, I think it can be closed.